src/Sound/ADPCMCodec.class.st

```diff
@@ -25,10 +25,12 @@ Class {
 		'stepSizeTable',
 		'indexTable'
 	],
-	#category : #'Sound-Core'
+	#category : 'Sound-Core',
+	#package : 'Sound',
+	#tag : 'Core'
 }
 
-{ #category : #'instance creation' }
+{ #category : 'instance creation' }
 ADPCMCodec class >> new [
```

Comment:
This is not needed

Reply:
Yes, i know! But, were did this come from? I committed only SoundBuffer in Pharo 12 within Iceberg.

Reply:
This is because the way method are categorized changed between P11 and P12 so it produces large diffs of changes. We could not make it differently.

Reply:
Thanks, for this explanation.